artemis/modules/utils/wappalyzer/main.go

```diff
@@ -13,10 +14,16 @@ import (
 	wappalyzer "github.com/projectdiscovery/wappalyzergo"
 )
 
+var client = &http.Client{
+	Transport: &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	},
+}
+
 func scan(url string, wappalyzerClient *wappalyzer.Wappalyze) map[string][]string {
 	result := make(map[string][]string)
 
-	resp, err := http.DefaultClient.Get(url)
+	resp, err := client.Get(url)
```

Comment:
would you mind adding a test? e.g that technology identification on https://self-signed.badssl.com/ works?